TwilioAuth/1.2.0/TwilioAuth.podspec

```diff
@@ -2,7 +2,10 @@ Pod::Spec.new do |s|
   s.name         = "TwilioAuth"
   s.version      = "1.2.0"
   s.summary      = "Twilio Auth SDK"
-  s.description  = "The TwilioAuth SDK lets you embed Time-based One Time Passwords (TOTP) or OneTouch (Push) authenticators into your mobile application. Your apps can then be used to generate TOTP codes for login or approve logins via push."
+  s.description  = %@The TwilioAuth SDK lets you embed Time-based One Time Passwords (TOTP) or OneTouch (Push) authenticators 
```

Comment:
Is the `%@ ... @` intended?

Reply:
yes, that way the line won't get too far in the file. But it doesn't change the outcome of the description